.travis.yml

```diff
@@ -53,3 +53,4 @@ install:
 script:
   - py.test om4labs
   - black --check om4labs
+  - cd testing; make all
```

Comment:
this will fail because travis won't let you copy that much data over

Reply:
It sort of worked ...